packages/base/package.json

```diff
@@ -22,23 +22,23 @@
     "test": "eslint ."
   },
   "devDependencies": {
-    "@metamask/auto-changelog": "^3.0.0",
-    "eslint": "^8.45.0",
-    "eslint-config-prettier": "^8.5.0",
+    "@metamask/auto-changelog": "^3.4.4",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "~2.26.0",
-    "eslint-plugin-jsdoc": "^41.1.2",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-jsdoc": "^47.0.2",
+    "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-promise": "^6.1.1",
-    "prettier": "^2.7.1"
+    "prettier": "^3.2.5"
   },
   "peerDependencies": {
-    "eslint": "^8.27.0",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "~2.26.0",
-    "eslint-plugin-jsdoc": "^39.6.2 || ^41 || ^43.0.7",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-jsdoc": "^47.0.2",
+    "eslint-plugin-prettier": "^5.1.3",
```

Comment:
I added a note at the end, but I take this back — the major change here is a reliance on Prettier 3. If we want to not make that change yet, then perhaps we ought to roll this back.

Reply:
To answer your comment though:

> If there's nothing major, why force users to update rather than maintaining compatibility with `eslint-plugin-prettier` v4?

In this case, if there is nothing that clients have to do in upgrading to Prettier 3, then why _not_ ask users them to upgrade? I would think that we would want to keep up to date on the latest versions of our tooling, no?

Reply:
Okay, sure, fair enough.